tribe-ext-community-currency-symbol/index.php

```diff
@@ -0,0 +1,415 @@
+
+		if ( 'prefix' === $default_position ) {
+			$opposite = 'suffix';
+		} else {
+			$opposite = 'prefix';
+		}
+
+		$symbol_position_list_array[ $opposite ] = $this->get_symbol_position_phrase( $opposite );
+
+		$current_position = (string) $this->get_current_symbol_position();
+
+		$post_id = Tribe__Main::post_id_helper();
+
+		// Awaiting https://central.tri.be/issues/80685 instead of the empty() check here
+		if ( '' === $current_position && empty( $post_id ) ) {
+			$current_position = $default_position;
+		}
+
+		$output = sprintf( '%1$s<option value=""></option>%1$s', PHP_EOL );
+
+		foreach ( $symbol_position_list_array as $key => $value ) {
+			$selected = selected( $key, $current_position, false );
+
+			$opt_text = esc_attr_x( $value, 'Currency symbol position', 'tribe-extension' );
+
+			$output .= sprintf( '<option value="%s" %s>%s</option>%s', esc_attr( $key ), esc_html( $selected ), $opt_text, PHP_EOL );
+		}
+
+		return $output;
+	}
+
+	/**
+	 * Detect which mode of the Community Events form we are currently in.
+	 *
+	 * @param null $page
+	 *
+	 * @return bool
+	 */
+	protected function is_community_page( $page = null ) {
+		$main = tribe( 'community.main' );
+
+		$event_post_type = Tribe__Events__Main::POSTTYPE;
+		$action          = Tribe__Utils__Array::get( $main->context, 'action' );
+		$ce_post_type    = Tribe__Utils__Array::get( $main->context, 'post_type', $event_post_type ); // assume event post type if not set
+
+		switch ( $page ) {
+			case 'add-event':
+				if (
+					'add' === $action
+					&& $event_post_type === $ce_post_type
+				) {
+					$is_page = true;
+				}
+				break;
+
+			case 'edit-event':
+				if (
+					'edit' === $action
+					&& $event_post_type === $ce_post_type
+				) {
+					$is_page = true;
+				}
+				break;
+
+			default:
+				$is_page = false;
+				break;
+		}
+
+		return $is_page;
+	}
+
+	/**
+	 * Do the output
+	 *
+	 * @return string
+	 */
+	public function display() {
+		$symbol_list = $this->symbol_list_select_options();
+
+		// if no symbols are entered in Settings, do not display Symbol or Position on CE form
+		if ( empty( $symbol_list ) ) {
+			return '';
+		}
+
+		?>
+
+		<table class="tribe-section-content">
+			<colgroup>
+				<col class="tribe-colgroup tribe-colgroup-label">
+				<col class="tribe-colgroup tribe-colgroup-field">
+			</colgroup>
+
+			<tr class="tribe-section-content-row">
+				<td class="tribe-section-content-label">
+					<?php tribe_community_events_field_label( 'EventCurrencySymbol', esc_html__( 'Currency Symbol:', 'tribe-extension' ) ); ?>
+				</td>
+				<td class="tribe-section-content-field">
+					<select
+						id="EventCurrencySymbol"
+						aria-label="<?php esc_attr_e( 'Currency Symbol', 'tribe-extension' ); ?>"
+						name="EventCurrencySymbol"
+						class="event-cost-currency-symbol tribe-dropdown"
+						placeholder="<?php esc_attr_e( 'Select a currency symbol', 'tribe-extension' ); ?>"
+					>
+						<?php echo $symbol_list; ?>
```

Comment:
We should escape this right?

Reply:
@bordoni `$symbol_list` is the full HTML. Each `<option>` is escaped upstream in `symbol_list_select_options()` -- is this ok?

Reply:
If they are safe / the attributes and inner HTML was escaped upstream, that's gravy. Sometimes it can be handy to note that, ie:

```php
<?php echo $symbol_list; // Escaped upstread ?>
```

Reply:
duly noted